homeassistant/components/zwave/__init__.py

```diff
@@ -236,23 +236,22 @@ def nice_print_node(node):
     print("\n\n\n")
 
 
+@retry.retry(exc_type=RuntimeError)
```

Comment:
I think that this is a hack. We should instead make it thread safe.

If we want to hot fix this now, I suggest we do not add a global decorator but instead do a simple trick with adding tries=5 to the parameters.